test_scripts/Polices/Policy_Table_Update/055_ATF_P_TC_HMI_sends_GetURLs_one_app_registered_urls_3default.lua

```diff
@@ -0,0 +1,125 @@
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local testCasesForPolicyTableSnapshot = require('user_modules/shared_testcases/testCasesForPolicyTableSnapshot')
+
+--[[ General Precondition before ATF start ]]
+commonSteps:DeleteLogsFileAndPolicyTable()
+testCasesForPolicyTable.Delete_Policy_table_snapshot()
+testCasesForPolicyTable:Precondition_updatePolicy_By_overwriting_preloaded_pt("files/jsons/Policies/Policy_Table_Update/few_endpoints_appId_default.json")
+
+--ToDo: shall be removed when issue: "ATF does not stop HB timers by closing session and connection" is fixed
+config.defaultProtocolVersion = 2
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+function Test:Precondition_trigger_getting_device_consent()
+  testCasesForPolicyTable:trigger_getting_device_consent(self, config.application1.registerAppInterfaceParams.appName, config.deviceMAC)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_PTU_GetURLs()
+  local endpoints = {}
+  testCasesForPolicyTableSnapshot:extract_pts(
+    {config.application1.registerAppInterfaceParams.appID},
+    {self.applications[config.application1.registerAppInterfaceParams.appName]})
+
+  for i = 1, #testCasesForPolicyTableSnapshot.pts_endpoints do
+    if (testCasesForPolicyTableSnapshot.pts_endpoints[i].service == "0x07") then
+      endpoints[#endpoints + 1] = {
+        url = testCasesForPolicyTableSnapshot.pts_endpoints[i].value,
+        appID = testCasesForPolicyTableSnapshot.pts_endpoints[i].appID}
+    end
+  end
+
+  local RequestId = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+
+  EXPECT_HMIRESPONSE(RequestId,{result = {code = 0, method = "SDL.GetURLS"} } )
+  :Do(function(_,data)
+      local is_correct = {}
+      for i = 1, #data.result.urls do
+        is_correct[i] = false
+        for j = 1, #endpoints do
+          if ( data.result.urls[i].url == endpoints[j].url ) then
+            is_correct[i] = true
+          end
+        end
+      end
+      if(#data.result.urls ~= #endpoints ) then
+        self:FailTestCase("Number of urls is not as expected: "..#endpoints..". Real: "..#data.result.urls)
+      end
+      for i = 1, #is_correct do
+        if(is_correct[i] == false) then
+          self:FailTestCase("url: "..data.result.urls[i].url.." is not correct. Expected: "..endpoints[i].url)
+        end
+      end
+    end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_PTU_GetURLs_AppRegistered()
```

Comment:
@dboltovskyi, the step checks results in Local PT, as this is not in scope of BB testing step is removed from the scripts 055, 056, 057